Tests/Command/ConsumeCommandTest.php

```diff
@@ -16,28 +20,93 @@ class ConsumeCommandTest extends KernelTestCase
 
     protected $mockConsumer;
 
-    public function setMockConsumer($expirationCount)
+    /**
+     * @var MockObject
+     */
+    private $mockQueueDriver;
+
+    public function setUp()
     {
         self::bootKernel();
+    }
+
+    public function setMockQueueDriver()
+    {
+        $msg = new QueueMessage(new Message());
+
+        $this->mockQueueDriver = $this
+            ->getMockBuilder(QueueDriverInterface::class)
```

Comment:
By default, when a mock is built, setups dummy methods for all the methods of the class. If we don't need do give a special behaviour to the method, I think is not needed to specify all the methods and we could have something like this
```
$this->mockQueueDriver = $this->createMock(QueueDriverInterface::class)
```

Reply:
I removed the driver mock as it seems to make no difference to have this in this class or not and refactored the consumer mock to the earlier state.